modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,81 @@
+= Import a TLS Context from Secrets Manager (Advanced)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your environment has specific security requirements or requires advanced configuration, you can import a TLS context from the secrets manager.
```

Comment:
I understand this procedure to be used when there's a certificate that's already stored in the secrets manager. 

Unless there are additional scenarios that this method supports which are not supported with the direct cert import method from within the RTF UI, we could re-word this sentence accordingly.